CanHazFunny/CanHazFunny/Program.cs

```diff
@@ -1,10 +1,13 @@
 ﻿namespace CanHazFunny;
 
-class Program
+public  sealed class Program
 {
-    static void Main(string[] args)
+    public static void Main(string[] args)
     {
-        //Feel free to use your own setup here - this is just provided as an example
-        //new Jester(new SomeReallyCoolOutputClass(), new SomeJokeServiceClass()).TellJoke();
+
+        var OutputInstance = new JokeOutput();
+        var ServiceInstance = new JokeService();
+
```

Comment:
I don't see the reason for needing to initialize the output and the service when you could do it in the new Jester.
```suggestion
new Jester(new JokeOutput(), new JokeService()).TellJoke();
```